packages/core/src/card/Card.css

```diff
@@ -1,19 +1,70 @@
 /* Styles applied to the root element */
 .saltCard {
-  box-shadow: var(--saltCard-boxShadow, var(--salt-overlayable-shadow));
-  border-width: var(--salt-size-border);
+  border-width: var(--saltCard-borderWidth, var(--salt-size-border));
   border-style: var(--salt-container-borderStyle);
-  padding: var(--saltCard-padding, var(--salt-spacing-300));
+  padding: var(--saltCard-padding, var(--salt-spacing-200));
+  position: relative;
 }
 
 .saltCard-primary {
   background: var(--saltCard-background, var(--salt-container-primary-background));
   border-color: var(--salt-container-primary-borderColor);
+  --card-accentColor: var(--saltCard-accentColor, var(--salt-container-primary-borderColor));
 }
 
 .saltCard-secondary {
   background: var(--saltCard-background, var(--salt-container-secondary-background));
   border-color: var(--salt-container-secondary-borderColor);
+  --card-accentColor: var(--saltCard-accentColor, var(--salt-container-secondary-borderColor));
+}
+
+/* Styles applied if `hoverable` is set */
+.saltCard-hoverable:hover {
+  box-shadow: var(--saltCard-boxShadow-hover, var(--salt-overlayable-shadow-hover));
+}
+
+/* Styles applied to create accent */
+.saltCard-accent::after {
+  content: "";
```

Comment:
test on incoming rounded corner, to avoid repeated work

Reply:
what specifically needs to be tested? is it a case of adding radius to the root css?

Reply:
it is possible (with overflow:hidden) and design says they aren't considering rounded corners for this release of Card